exercises/exercises2.cpp

```diff
@@ -0,0 +1,67 @@
+#include "iterator"
+#include "iostream"
+#include "numeric"
+#include "thread"
+#include "vector"
```

Comment:
ten includy z stla to w nawiasach trójkątnych dawaj <>

Reply:
zadanie 3 wyniki:

Reply:
ps includy poprawione